frontends/ol-components/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from "react"
+import styled from "@emotion/styled"
+import { default as MuiAlert, AlertColor } from "@mui/material/Alert"
+import { theme } from "../ThemeProvider/ThemeProvider"
+import type { AlertProps as MuiAlertProps } from "@mui/material/Alert"
+import { pxToRem } from "../ThemeProvider/typography"
+
+type Colors = {
+  [Severity in AlertColor]: string
+}
+
+const COLORS: Colors = {
+  info: theme.custom.colors.blue,
+  success: theme.custom.colors.green,
+  warning: theme.custom.colors.orange,
+  error: theme.custom.colors.lightRed,
+}
+
+type AlertStyleProps = {
+  severity: AlertColor
+}
+
+const AlertStyled = styled(MuiAlert)<AlertStyleProps>(({ severity }) => ({
+  padding: "11px 16px",
+  borderRadius: 4,
+  borderWidth: 2,
+  borderStyle: "solid",
+  borderColor: COLORS[severity],
+  background: "#FFF",
+  ".MuiAlert-message": {
+    lineHeight: pxToRem(22),
+    verticalAlign: "middle",
+    fontSize: pxToRem(14),
```

Comment:
The designs say this is `P2` (@steven-hatch hey i thought we were going to change that to body1 body2 body3...) so I would do here:
```
...theme.typography.body2,
```
rather than setting lineHeight and fontSize

Reply:
Now using `theme.typography.body2`, but this doesn't have any line height at present - the text is a pixel off vertically and perhaps best not to position independently if it should be controlled by the theme.

Reply:
Yeah, that's a mistake. The typography styles should specify the line height. I added them for subtitles and headings in #865, I'll add the body line heights, too.

But `theme.typography.body2` specify the vertical-alignment middle, so if you need that, add it in the component.